scss/_card.scss

```diff
@@ -7,7 +7,7 @@
   display: flex;
   flex-direction: column;
   background-color: $card-bg;
-  border: $card-border-width solid $card-border-color;
+  box-shadow: $card-shadow;
```

Comment:
from my gutfeeling I would have used the variables, if this does not create any problems.